src/View/Helper/FormHelper.php

```diff
@@ -1167,9 +1170,14 @@ protected function _optionsOptions($fieldName, $options)
             return $options;
         }
 
+        if (substr($fieldName, -5) === '._ids') {
+            $fieldName = substr($fieldName, 0, -5);
+        } elseif (substr($fieldName, -3) === '_id') {
+            $fieldName = substr($fieldName, 0, -3);
+        }
```

Comment:
Why is this necessary? Line 1170 currently takes the first segment which would be `tags` in the case of `tags._ids`.